### PR TITLE
Updated URLs in line with API changes for LT-998

### PR DESCRIPTION
**Pull request by andytaylorkainos**

### CODE REVIEW

File: seed_data/check_documents.py [hunk 1]
```diff
@@ -1,5 +1,6 @@
 from ..seed_data.make_requests import make_request
-from ..tools.wait import wait_for_ultimate_end_user_document, wait_for_third_party_document, wait_for_additional_document, wait_for_document
+from ..tools.wait import wait_for_ultimate_end_user_document, wait_for_third_party_document, \
```
Comment (craiga) [inline seed_data/check_documents.py]: At this point, would it make more sense just to import the `wait` module?

File: seed_data/check_documents.py [hunk 2]
```diff
@@ -8,23 +9,25 @@ def check_document(url, base_url, export_headers):
 
 
 def check_end_user_document_is_processed(draft_id, base_url, export_headers):
-    return check_document('/drafts/' + draft_id + '/end-user/document/', base_url, export_headers)
+    return check_document('/applications/' + draft_id + '/end-user/document/', base_url, export_headers)
```
Comment (craiga) [inline seed_data/check_documents.py]: Is `draft_id` still an appropriate name here?

File: seed_data/seed_data.py [hunk 3]
```diff
@@ -42,12 +42,16 @@ def __init__(self, seed_data_config):
         self.seed_good.add_good()
 
         self.seed_clc = SeedClc(self.base_url, self.gov_headers, self.export_headers, self.request_data, self.context)
-        self.seed_party = SeedParty(self.base_url, self.gov_headers, self.export_headers, self.request_data, self.context)
+        self.seed_party = SeedParty(self.base_url, self.gov_headers, self.export_headers, self.request_data,
+                                    self.context)
```
Comment (craiga) [inline seed_data/seed_data.py]: Why the sudden wrapping? Do we have an agreed preferred line length?

File: seed_data/seed_data.py [hunk 4]
```diff
@@ -58,20 +62,21 @@ def add_to_context(self, name, value):
...
-                     body={'sites': [self.context['primary_site_id']]})
+        make_request("POST", base_url=self.base_url, url='/applications/' + draft_id + '/sites/',
+                     headers=self.export_headers, body={'sites': [self.context['primary_site_id']]})
 
     def create_draft(self, draft):
         self.log("Creating draft: ...")
-        data = self.request_data['draft'] if draft is None else draft
-        response = make_request("POST", base_url=self.base_url, url='/drafts/', headers=self.export_headers, body=data)
-        draft_id = response.json()['draft']['id']
+        data = self.request_data['application'] if draft is None else draft
```
Comment (craiga) [inline seed_data/seed_data.py]: Alternatively you could treat `None` as false-ish and shorten this a bit:

```python
data = draft if draft else self.request_data['application']
```

File: seed_data/seed_data.py (hunk 3 above)
Comment (andytaylorkainos) [reply seed_data/seed_data.py]: 120 :)

File: seed_data/check_documents.py (hunk 2 above)
Comment (andytaylorkainos) [reply seed_data/check_documents.py]: We're still trying to distinguish between the state of the application (whether it is in draft, or if it has been submitted)

**Pull request merged**